projects/magento/tests/GiftCardPayment.spec.js

```diff
@@ -15,7 +15,8 @@ const giftCard50EUR = paymentResources.giftCard.EUR50;
 const giftCard20EUR = paymentResources.giftCard.EUR20;
 const user = paymentResources.guestUser.dutch;
 
-test.describe.parallel("Payment via Gift Card", () => {
+// TODO
```

Comment:
```suggestion
```

Reply:
Hey @hossam-adyen it's a fair suggestion, however this PR will be merged to develop-9, which is a WIP branch, and a few things marked with .fixme rely on fixes to the actual product. We will keep working on this WIP branch alongside the magento v9 relese. There are follow up tickets for more fixes.